portfolio-app/app/theme/ThemeProvider.tsx

```diff
@@ -0,0 +1,25 @@
+"use client"; // Ensure this is a client component
+
+import { ReactNode } from 'react';
+import { ThemeProvider as MUIThemeProvider, createTheme } from '@mui/material/styles';
+
+const themeOptions = {
```

Comment:
Nice I didnt mention this for a reason. I am glad you were able to figure this part out. This can become a very important tool for styling as well. I will teach you more about this file later on. Well done